pkgs/applications/video/filebot/default.nix

```diff
@@ -9,11 +9,11 @@ let
   };
 in stdenv.mkDerivation rec {
```

Comment:
I know that what I'm going to ask is out of the scope of the this PR but do you think you could use the `finalAttrs` pattern here?

Let me share a couple of useful links regarding this:

1. History: https://discourse.nixos.org/t/avoid-rec-expresions-in-nixpkgs/8293
2. Documentation: https://nixos.org/manual/nixpkgs/unstable/#mkderivation-recursive-attributes
3. Recent example of implementation: https://github.com/NixOS/nixpkgs/compare/17f96f7b978e61576cfe16136eb418f74fab9952..9e6ea843e473d34d4f379b8b0d8ef0425a06defe

Reply:
Interesting, I didn't know about that. Thanks for the hint. 🙂